electrum/plugins/ledger/qt.py

```diff
@@ -1,6 +1,6 @@
 from functools import partial
 
-#from btchip.btchipPersoWizard import StartBTChipPersoDialog
```

Comment:
This line was commented out on purpose.
`StartBTChipPersoDialog` was only used from unreachable code, and it was made unreachable because `btchip.btchipPersoWizard` would need updating for it to work. (specifically `btchip.btchipPersoWizard` used PyQt4, and we migrated to PyQt5 around 2 years ago)

I have now removed this stuff in https://github.com/spesmilo/electrum/commit/f405c3fbddc813837403f7cfae3ccb6c1145f838